### PR TITLE
Darrell countries

### PR DESCRIPTION
**Pull request by LavishGent**
Created a Countries Application using a documented API to then populate the page with the appropriate information. Features include the ability to search, the ability to filter by region, and view one country. Clicking on one of the border countries navigates you to a single view. Also you can toggle between light and dark themes. CSS was based on the provided style guide and the application is mobile-friendly.

### CODE REVIEW

**Comment by amber-ensley** [comment]
Hey Darrell! 

You met all your requirements, and your code looks good. Here are some tips to help you in the future: 

HTML
not too hard to follow. You can add comments as labels to section off your html 
name your functions in relation to your action you're trying to accomplish
look into BEM classname style 

CSS
good use of variables
good use of flex
follow hierarchy of universal, body, tags, class, and id selectors

JS
have a separate folder for utils
look into ternary operators and truthy/falsey values

Other than that, good job!

**Comment by kyanethmoran** [comment]
html: 
	clean html with no unnessecary code snippets such as unused classes and id's
	structural flow is very easy to follow and doesnt lend itself to becoming a puzzle
	naming convention are easy to follow and undestandable. very clear and concise use 
	of different containers that really make the code legable and structurally sound
	for understanding the flow and content blocks

javascript:
	textboxes used to populate html structure and data are clear and very easy to read. 
	using the ${value.flag} syntax directly in the contentboxes makes reading the code 
	alot easier than having to refer back to variables 
	
css:
	css follows top down from the layout of the page which makes picking apart the bits
	alot easier. pixels are typically used with pixel based things like boarders and 
	other type of measures like em are used for content boxes or areas
